moonbeam-types-bundle/index.ts

```diff
@@ -680,7 +691,7 @@ export const moonbeamDefinitions = {
           state: "CollatorStatus",
         },
         NominatorAdded: {
-          _enum: ["AddedToBottom", { AddedToTop: "Balance" }],
+          _enum: [{ AddedToTop: "Balance" }, "AddedToBottom"],
```

Comment:
why did this change

Reply:
Subscan reported that the order of the enum variants matters. The new order is the same order that the Rust struct uses. This change prevents some decoding issues that we probably did not try to do ourselves because the object is only used in certain events (not used in storage at all)